### PR TITLE
Random personalities

### LINKED ISSUE #194: Disconnect AI names and personalities
### I have an idea that might make the game better...
Picking an AI name will randomly generate the personality.

### PR DESCRIPTION
**Pull request by vincentcote87**
Fixes #194
#### Overview of changes

- Picking an AI will generate random personality.

Reviewer: @johnanvik

### CODE REVIEW

File: src/classes/Personality.js [hunk 1]
```diff
@@ -2,16 +2,19 @@ import Sprinter from './Sprinter'
...
+  constructor() {
+    let num = Math.floor((Math.random() * 3) + 1);
+    switch(num) {
+      case 1:
+      case 2: this.personality = new Sprinter(); break;
+      case 3: this.personality = new Hacker(); break;
     }
-    if(name === 'Joker') {
-      this.personality = new Hacker();
+    if(num === 1) {
```
Comment (johnanvik) [inline src/classes/Personality.js]: Move this into the switch, just don't use the break.

File: src/components/MainComponent.vue [hunk 2]
```diff
@@ -210,15 +209,8 @@ export default {
     this.$store.commit('setGameState', {gameState: 'startPlayerTurn'})
 
   },
-  updated() {
-      this.deleteData.push(this.$store.state.currentGameState);
-  }
 //  updated() {
```
Comment (johnanvik) [inline src/components/MainComponent.vue]: Remove commented code

File: src/classes/Personality.js (hunk 1 above)
Comment (vincentcote87) [inline src/classes/Personality.js]: oh yea that makes much more sense. thanks.

**Comment by vincentcote87** [comment]
changes made.

**Pull request merged**